demos/mailer/README.md

```diff
@@ -1,5 +1,7 @@
 # mailer plugin
 
+Requires `mailer-plugin` >= 1.22 *(earilier versions won't allow `smtpHost` to be set)*
```

Comment:
```suggestion
Requires `mailer-plugin` >= 1.22
```
You can skip the reasoning imho :)

Reply:
okay, just thought it would be good to know why the requirement exists ;)

Reply:
currently, other demos haven't specified why so :)

Reply:
I wonder @oleg-nenashev if we should add optional dependencies to force users to update to our recommended/required plugins.

Reply:
Please don't.
We could add an administrative monitor for JCasC compatibility advises.
But adding an optional dependency is a can of worms for downstream dependencies

Reply:
True 😆